File: Web/JavaScriptPrograms/System Components Checker.js
var os = require('os');
if (os.type() === "Windows_NT") {
  system = "Windows";
}

else {
  system = os.type()
}
console.log("System: " + system)
console.log("Node Name: " + os.hostname())
console.log("Version: " + os.release())
console.log("Machine: " + os.arch())
console.log("Current Working Directory: " + process.cwd())

File: Python/PythonPrograms/SystemComponentsChecker.py
import platform
import os 

cwd = os.getcwd()
my_system = platform.uname()
 
print(f"System: {my_system.system}")
print(f"Node Name: {my_system.node}")
print(f"Version: {my_system.version}")
print(f"Machine: {my_system.machine}")
print(f"Current Working Directory: {cwd}")